parser.js

```diff
@@ -85,11 +82,11 @@ function setData(){
 
 	
 	var tempString="";
-	tempString=capFirst(tempData[0]);
+	tempString=tempData[0];

```

Comment:
what do you mean make a constant? you're refering to the the 0 right? 
so instead of the 0 put a variable that says what it is...correct? 
so like var FirstThing=0?